invenio_rdm_records/jsonschemas/records/record-v1.0.0.json

```diff
@@ -123,19 +123,152 @@
 
       }
     },
-    "_created": {
+    "dates": {
+      "description": "Date interval.",
+      "items": {
+        "additionalProperties": false,
+        "properties": {
+          "description": {
+            "description": "Description of the date interval.",
+            "type": "string"
+          },
+          "end": {
+            "description": "End date.",
+            "type": "string",
+            "format": "date-time"
+          },
+          "start": {
+            "description": "Start date.",
+            "type": "string",
+            "format": "date-time"
+          },
+          "type": {
+            "description": "Type of the date interval."
+          }
+        },
+        "required": [
+          "type"
+        ],
+        "type": "object"
+      },
+      "type": "array"
+    },
+    "description": {
+      "description": "Description/abstract for record.",
+      "type": "string"
+    },
+    "embargo_date": {
+      "description": "Embargo date of record (ISO8601 formatted date).",
+      "title": "Embargo Date",
```

Comment:
remove "title"

Do we keep string with format "date-time" or switch to `"type": "datetime"` ?

Reply:
I do not find the JSONSchema `datetime` type. I am looking [here](https://json-schema.org/understanding-json-schema/reference/type.html) could you point me to the source?

Reply:
Discussed IRL, it's ok as it is.

Reply:
my bad, was thinking about `format`